proto/decentraland/renderer/kernel_services/sign_request.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package decentraland.renderer.kernel_services;
+
+enum RequestMethod {
+    GET = 0;
+    POST = 1;
+    PUT = 2;
+    PATCH = 3;
+    DELETE = 4;
+}
+
+message SignBodyResponse {
```

Comment:
I would rather suggest this as response:
```proto
message SignBodyResponse {
    repeated string auth_chain = 1;
    uint64 timestamp = 2;
    string metadata = 3;
}
```

Reply:
done